requirements/base.txt

```diff
@@ -9,8 +9,8 @@ natsort >= 8, < 9
 netcdf4 >= 1.2, < 1.7
 numpy >= 1, < 2
 pandas >= 2.1.3, < 2.3  # Minimum bound is 2.1.3 because of a regression in v2.1.0/2.1.1 inflating time/memory consumption on groupby operations with MultiIndex
-pyomo >= 6.5, < 6.7.2
+pyomo >= 6.8.2
```

Comment:
Inkeeping with our current approach to set caps, can you set one for v7?

Reply:
I'll do so... but reluctantly. If the upperbound was not set, this issue would be invisible to newer users.
We really should move away from this, IMO.

Reply:
In reality, people do not really follow semantic versioning, and most python devs do not have the resources to support more than the current version of a tool...

Reply:
> If the upperbound was not set, this issue would be invisible to newer users.

Alternatively, if the Gurobi upper bound _had_ been set, this would still be invisible to new users ;)